### ISSUE TITLE
配置文件修改数据集

### ISSUE THREAD

**Issue by ff12343** [PaddlePaddle/PaddleOCR#6962]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

rec_svtrnet.yml 配置文件中数据集部分为LMDBDataSet，想修改为SimpleDataSet，如何修改参数

原始参数如下：
Train:
  dataset:
    name: LMDBDataSet
    data_dir: ./train_data/
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          character_dict_path:./model/svtr_rec_model/rec_svtr_tiny_none_ctc_en_train/ZG_dict.txt
          image_shape: [3, 64, 256]
          padding: False
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 512
    drop_last: True
    num_workers: 4
请问修改哪些参数，可以使配置文件支持SimpleDataSet

**Comment by zhangxing1209** [comment]
参考ch_PP-OCRv3_rec_distillation.yml 就可以了
